packages/vm/tests/BlockchainTestsRunner.ts

```diff
@@ -119,14 +119,18 @@ export default async function runBlockchainTest(options: any, testData: any, t:
       common.setHardforkByBlockNumber(currentBlock.toNumber())
 
       if (raw.transactionSequence) {
+        const parentBlock = await vm.blockchain.getIteratorHead()
+        const blockBuilder = await vm.buildBlock({
+          parentBlock,
+          headerData: { coinbase: '0x96dc73c8b5969608c77375f085949744b5177660' },
+          blockOpts: { calcDifficultyFromHeader: parentBlock.header, freeze: false },
+        })
         for (const txData of raw.transactionSequence) {
           const txRLP = Buffer.from(txData.rawBytes.slice(2), 'hex')
           const tx = TransactionFactory.fromSerializedData(txRLP)
           const shouldFail = txData.valid == 'false'
```

Comment:
looks like the `blockchain:london` tests are having some `expectException: TR_TipGtFeeCap` failures here, so I am wondering if `shouldFail` should also take `expectException` into account

Reply:
e.g. [here](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1374/checks?check_run_id=3137135710#step:7:57872):
<img width="1143" alt="Screen Shot 2021-07-22 at 1 15 38 PM" src="https://user-images.githubusercontent.com/22116/126703872-d9809bea-e281-4982-a285-84789051796c.png">
```
expectException:
        ">=London": TR_TipGtFeeCap
```

Reply:
That should be a new PR. We indeed have some tests which pass, but they only pass because they throw on something else, and not for the expected reason. However this means we have to inspect each error string and figure out what that means. But on the other hand, we want to ensure that the tests fail for the right reason. This error is fixed now.